crates/dbsp/benches/column_layer.rs

```diff
@@ -33,7 +33,7 @@ where
     let mut data = LeanVec::with_capacity(length);
 
     for _ in 0..length {
-        let (k, w) = rng.gen();
+        let (k, w) = rng.r#gen();
```

Comment:
What is this?

Reply:
`gen` is a keyword in Rust 2024, but not in Rust 2021. An `r#` prefix allows you to use anything as an identifier even if it would otherwise be a keyword.